examples/basic_tutorial_BERT.ipynb

```diff
@@ -0,0 +1,1198 @@
+{
```

Comment:
<p>the comment here , I think, needs to be corrected: <code>raw</code> -&gt; <code>mean aggregated</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/giotto-ai/giotto-deep/pull/114/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>You are absolutely right. Fixed.</p>